src/Bicep.Cli/Logging/BicepDiagnosticLogger.cs

```diff
@@ -45,7 +46,7 @@ public void LogSummary()
             this.logger.Log(ToLogLevel(DiagnosticLevel.Info), summary);
         }
 
-        public int ErrorCount { get; private set; }
+        public int ErrorCount { get; set; }
```

Comment:
>set [](http://example.com/codeflow?start=37&length=3)

Why does error count need to be settable from outside this class?

Reply:
Good spot! This needs reverting back to a private setter.